.github/workflows/CI.yml

```diff
@@ -26,15 +26,22 @@ jobs:
           python-version: "${{ matrix.python-version }}"
       - name: Setup GNU Fortran
         uses: modflowpy/install-gfortran-action@v1
+      - name: Install MODFLOW 6
+        uses: modflowpy/install-modflow-action@v1
+        with:
+          repo: modflow6
```

Comment:
```suggestion
```
I would prefer to either test against latest or nightly.
Nightly makes it easier to synchronize changes with MODFLOW so I think we should use nightly

Reply:
I see benefits of testing both a stable release and development snapshot. Testing only against nightly can be unstable, as you are aware. But it's also important to ensure new features with xmipy work with existing releases of mf6 libraries. The extra installation and tests adds approx. 15 seconds to each CI job, which is minimal. Coverage reports are appended too.

Reply:
> But it's also important to ensure new features with xmipy work with existing releases of mf6 libraries

We sometimes make breaking changes like this one https://github.com/Deltares/xmipy/pull/98
Continuing to test against stable, means I have to wait until a new modflow release is out before I can merge a breaking PR. However, those don't happen that often, so let's give it a try. 

However, please parametrize stable and nightly rather than running them sequentially.

Reply:
sure, parameterized with `modflow6-repo`